cmd/jiralert/main.go

```diff
@@ -154,6 +154,7 @@ func setupLogger(lvl string, fmt string) (logger log.Logger) {
 	case "debug":
 		filter = level.AllowDebug()
 	case "info":
+		fallthrough
```

Comment:
You could have just dropped the `case "info"` instead and relied on the `default`.